README.md

```diff
@@ -0,0 +1,4 @@
+# Ecosia Privacy Policy 
+[![GitHub license](https://img.shields.io/badge/License-CC%20BY--NC--ND%204.0-yellow.svg)](https://github.com/ecosia/ecosia-privacy-policy/blob/master/LICENSE)
+
+This repository aims to store Ecosia's Privacy Policy in multiple languages, using markdown
```

Comment:
- I'd move the mention of markdown in another sentence: 
> […] in multiple languages. The policy files are written in Markdown and their names follow the pattern `{languageCode}.md`. 
- I'd add a `##` section to specify contact pointers in case users have questions

Reply:
@AndreGCRamos Up, whenever you have some time ;)